homeassistant/components/flo/sensor.py

```diff
@@ -20,8 +21,10 @@
 NAME_DAILY_USAGE = "Today's Water Usage"
 NAME_CURRENT_SYSTEM_MODE = "Current System Mode"
 NAME_FLOW_RATE = "Water Flow Rate"
-NAME_TEMPERATURE = "Water Temperature"
+NAME_WATER_TEMPERATURE = "Water Temperature"
+NAME_AIR_TEMPERATURE = "Temperature"
```

Comment:
Leak detector pucks measure *air* temperature, not water temperature. `FloTemperatureSensor` has been updated to handle both.